docs/en/administration/sync_accounts_between_multiple_hosts.md

```diff
@@ -46,9 +46,9 @@ Create an empty directory `account-sync` , save below content in `play.yaml` und
 
 
 
-Create a file named `hosts` in this directory, place IP addresses of all hosts need to create account in this file, each line with a host's IP.
+Create a file named `hosts` in this directory. Place IP addresses of all the hosts that need to create account in this file with one host's IP per line.
```

Comment:
I think it'd be better with:

> Create the Ansible inventory file `hosts`, which contains hostname / IP address for all hosts that need to create account.

there're examples below, I won't worry too much about the "IP per line" part.